timed/subscription/views.py

```diff
@@ -24,10 +23,8 @@ class SubscriptionProjectViewSet(viewsets.ReadOnlyModelViewSet):
     )
 
     def get_queryset(self):
-        queryset = Project.objects.annotate(
-            count_packages=Count('billing_type__packages')
-        )
-        return queryset.filter(archived=False, count_packages__gt=0)
+        return Project.objects.filter(
+            archived=False, customer_visible=True)
```

Comment:
a small nitpick: brace should be indented on the next line... it is more consistent to rest of module and in my opinion easier editable... ;).